File: www/tests/cgi-bin/Perl/variables.pl
#!/usr/bin/perl
print "Content-type: text/html\n\n";
print "<html><body>";
print "<h1>Variables de entorno CGI</h1>";
print "<table border='1'>";
foreach my $key (sort keys %ENV) {
    print "<tr><td>$key</td><td>$ENV{$key}</td></tr>";
}
print "</table>";
print "</body></html>";

File: www/web3/cgi-bin/Python/variables.py
#!/usr/bin/env python
import os

print("<html><body>")
print("<h1>Variables de entorno CGI</h1>")
print("<ul>")
for key, value in os.environ.items():
    print(f"<li><strong>{key}:</strong> {value}</li>")
print("</ul>")
print("</body></html>")